examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL/SDL.h>
```

Comment:
Ended up trying it on travis, getting internal errors though https://travis-ci.org/Cyberunner23/CITests/builds/135431759